lib/libutee/tee_api.c

```diff
@@ -83,6 +83,10 @@ TEE_Result TEE_InvokeTACommand(TEE_TASessionHandle session,
 	res = utee_invoke_ta_command(session, cancellationRequestTimeout,
 				      commandID, paramTypes, params,
 				      returnOrigin);
+	if (returnOrigin != NULL)
```

Comment:
The content of returnOrigin pointer can be tested only if returnOrigin is defined.